DM-36696: Make pex.config example docs be consistently sentences

bsmartradio opened the pull request:
Changed a few doc strings to be in sentence format/proper capitalization/punctation.

parejkoj reviewed (approved):
Thanks for this. There's one more in `registry-intro.rst` (it should have the kwargs named explicitly, too0.

bsmartradio commented:
> Thanks for this. There's one more in `registry-intro.rst` (it should have the kwargs named explicitly, too0.

Fixed the fragmented sentence and added the **kwargs arguments.
